.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+### 🛠 Summary
+
+JIRA/Issue if applicable
+Describe the changes
+
+### 🧪 Checklist
+
+- [ ] Functional tests done.
```

Comment:
Does it really mean that CI passed? We already have CI green mark for that.

I think it means that functional tests are in place in ovms-test. Which in 99% cases is false because our functional tests follow implementation. @mzegla

Reply:
In such case it's also out of scope for external contributors.